### PR TITLE
Add clan tag colour support

### PR DESCRIPTION
**Pull request by HardlySoftly**
Now including migration support

### CODE REVIEW

File: migrations/add_clan_tag_colors-150bb43b63f44549c415ae56853632f8779f8cd4.sql [hunk 1]
```diff
@@ -0,0 +1 @@
+ALTER TABLE clan_list ADD COLUMN `clan_tag_color` varchar(30) DEFAULT NULL COMMENT 'Color of the clan''s tag in chat';
```
Comment (micheljung) [inline migrations/add_clan_tag_colors-150bb43b63f44549c415ae56853632f8779f8cd4.sql]: Either add `COMMENT` in the db-structure as well, or remove it here... consistency :-)

Comment (micheljung) [inline migrations/add_clan_tag_colors-150bb43b63f44549c415ae56853632f8779f8cd4.sql]: And good question from Brutus, why `varchar(30)` and not `varchar(6)` (hex code)?

**Comment by Brutus5000** [comment]
VARCHAR(30) can be anything like "red-like-my-enemies-blood"
I propose VARCHAR(6) for a RGB-Hexcode.

**Comment by HardlySoftly** [comment]
I was thinking along the lines of human readable colour names http://htmlcolorcodes.com/color-names/

**Pull request merged**